Source/ACE.Server/Entity/Tailoring.cs

```diff
@@ -500,7 +500,8 @@ public static void UpdateCommonProps(Player player, WorldObject source, WorldObj
         {
             player.UpdateProperty(target, PropertyInt.PaletteTemplate, source.PaletteTemplate);
             //player.UpdateProperty(target, PropertyInt.UiEffects, (int?)source.UiEffects);
-            player.UpdateProperty(target, PropertyInt.MaterialType, (int?)source.MaterialType);
+            if (source.MaterialType.HasValue)
+                player.UpdateProperty(target, PropertyInt.MaterialType, (int?)source.MaterialType);
```

Comment:
can you include a description of the reasoning for this specific change, and a specific item / scenario from retail that demonstrates this?